crowdsec-docs/docs/intro.mdx

```diff
@@ -38,12 +47,39 @@ In addition to the core "detect and react" mechanism, CrowdSec is committed to s
 ## Architecture
 
 
+<!-- @kka : update schema:
+    - align text
+    - rename data sources to log sources
+    - move log sources out of the SE box
+-->
 <div style={{display: 'flex'}}>
     <div style={{textAlign: 'center', flex: '1'}}>
-    <img src={useBaseUrl('/img/crowdsec_architecture.png')}></img>
+    <img src={useBaseUrl('/img/simplified_SE_underthehood.png')}></img>
     </div>
 </div>
 
+Under the hood, the Security Engine has various components:
```

Comment:
Should we really use components here ?
We use components for things that are external to the SE code/binary

Let's use the word "Functions" or "Functionnal Processes" or something distinct indicating that it's PART of the binary and withing the SE code

Reply:
We might find something better than component, but neither "Functions" or "Functionnal Processes" convince me (as they're a bit misleading in my opinion)